policy/modules/admin/blueman.te

```diff
@@ -7,7 +7,6 @@ policy_module(blueman)
 
 type blueman_t;
 type blueman_exec_t;
-dbus_system_domain(blueman_t, blueman_exec_t)
```

Comment:
You'll need to add something back here, since `blueman_t` won't be a domain anymore if the dbus module  is removed.

Reply:
Thanks!